src/backend/executor/execAmi.c

```diff
@@ -730,8 +730,14 @@ ExecSquelchNode(PlanState *node)
 		return;
 
 	if (node->squelched)
+	{
```

Comment:
As we discussed, we need to add force(bool type) in ExecSquelchNode

Reply:
Fixed in the new commit.